build.gradle.kts

```diff
@@ -6,6 +6,8 @@ plugins {
 detekt {
     debug.set(true)
     isEnabled.set(true)
+    allRules.set(true)
+    parallel.set(true)
```

Comment:
Do we want these enabled? Gradle give us the parallelism and active all the rules seems like too much

Reply:
This was just for a test as it's on the example project that contains 5 lines of code.
I'm fine turning both of them off.